zerver/forms.py

```diff
@@ -460,7 +460,9 @@ def clean(self) -> Dict[str, Any]:
 
             if return_data.get("invalid_subdomain"):
                 logging.warning(
-                    "User %s attempted password login to wrong subdomain %s", username, subdomain
+                    "User attempted password login to wrong subdomain %s. Matching accounts: %s",
+                    subdomain,
+                    return_data.get("matching_user_ids_in_different_realms"),
```

Comment:
This was requested around here - https://github.com/zulip/zulip/pull/13924#issuecomment-915597341